TLSharp.Core/packages.config

```diff
@@ -1,3 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="MarkerMetro.Unity.Ionic.Zlib" version="2.0.0.14" targetFramework="net452" />
```

Comment:
if you rebased properly, then there shouldn't be any need of reintroducing this Ionic.Zlib dependency